justifi-tech/web-component-library
# #202 feat: Add payment-balance-transactions component

* Add payment-balance-transactions component

![image](https://github.com/justifi-tech/web-component-library/assets/22480988/70e3f364-01e7-47e8-a64e-536493f22b4e)

* Should be the same table displayed on the dashboard on "Payments" -> "Transactions" tab.

<img width="1672" alt="image" src="https://github.com/justifi-tech/web-component-library/assets/22480988/85f12753-2db6-40d0-8cc4-9e66ea604bb4">




Links
-----
[Issue #202](#202)



Developer considerations
--------------

  - If this is a MAJOR version, what are the breaking API changes? Was anything added to the API?
  - If this is a MINOR version, what feature was added? Why would consumers want this new version?
  - If this is a PATCH version, what was fixed?


Steps for Testing/QA
--------------------
* Pull the branch and run the project.
* Go to http://localhost:6006/?path=/story/components-paymentbalancetransactions--basic
* Add the `auth-token`, `payment-id` and `account-id` props
* The table with the transactions should be displayed
